bigquery/noxfile.py

```diff
@@ -40,6 +40,7 @@ def default(session):
 
     dev_install = ".[all]"
     session.install("-e", dev_install)
+    session.install("-e", os.path.join("..", "test_utils"))
```

Comment:
Would it make more add it to `LOCAL_DEPS` instead?  Then any test session could have it available.

Reply:
Well, perhaps someone might want to make use of that in other test sessions, thus why not, I'll move it.